src/Reader/Reader.spec.js

```diff
@@ -88,23 +88,25 @@ test('Reader runWith', t => {
 test('Reader ask errors', t => {
   const ask = bindFunc(Reader.ask)
 
-  t.throws(ask(undefined), TypeError, 'throws with undefined')
-  t.throws(ask(null), TypeError, 'throws with null')
-  t.throws(ask(0), TypeError, 'throws with falsey number')
-  t.throws(ask(1), TypeError, 'throws with truthy number')
-  t.throws(ask(''), TypeError, 'throws with falsey string')
-  t.throws(ask('string'), TypeError, 'throws with truthy string')
-  t.throws(ask(false), TypeError, 'throws with false')
-  t.throws(ask(true), TypeError, 'throws with true')
-  t.throws(ask([]), TypeError, 'throws with an array')
-  t.throws(ask({}), TypeError, 'throws with an object')
+  const err = /Reader.ask: Function required/
```

Comment:
Need to change the wording on this error to include language about no arguments.